shared/team-building/user-result.desktop.tsx

```diff
@@ -95,6 +96,8 @@ const Avatar = ({
 }) => {
   if (keybaseUsername) {
     return <Kb.Avatar size={AvatarSize} username={keybaseUsername} />
+  } else if (resultForService === 'keybase' || resultForService === 'contact') {
+    return <Kb.Avatar size={AvatarSize} username={'invalid username for placeholder avatar'} />
```

Comment:
No need for brackets around string literal, use double quotes

Reply:
I think this is copied from user-result.native, ill change there as well